src/classes/ATTD_CourseConnectionContact_TDTM.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>46.0</apiVersion>
```

Comment:
Do we want all new classes and trigger on 46.0 or do we want to wait until we move all metadata to 46.0?

Reply:
(Probably a larger question that we need to agree on as an entire team)

Reply:
Updated to 45.0 for now.